{{cookiecutter.project_slug}}/cmake/cache.cmake

```diff
@@ -0,0 +1,16 @@
+set(CACHE_OPTION "ccache" CACHE STRING "Compiler cache to be used.")
+set(CACHE_OPTION_VALUES "ccache" "sccache")
+set_property(CACHE CACHE_OPTION PROPERTY STRINGS ${CACHE_OPTION_VALUES})
+list(FIND CACHE_OPTION_VALUES ${CACHE_OPTION} CACHE_OPTION_INDEX)
+
+if(${CACHE_OPTION_INDEX} EQUAL -1)
+    message(STATUS "Using custom compiler cache system: ${CACHE_OPTION}, valid entries are ${CACHE_OPTION_VALUES}")
```

Comment:
Maybe `explicitly supported entries are`? The user could technically use any other executable name and hope it works.

Reply:
Done.